manual/dna-manual.bib

```diff
@@ -129,6 +129,14 @@ @article{csardi2006igraph
   url = {http://igraph.org},
 }
 
+@Manual{csardi2006remotes,
+  title = {remotes: R Package Installation from Remote Repositories, Including 'GitHub'},
+  author = {Csárdi, Gábor and Wickham, Hadley and Chang, Winston and Hester, Jim and Morgan, Martin and Tenenbaum, Dan},
+  year = {2018},
+  note = {R package version 2.0.2},
+  url = {https://CRAN.R-project.org/package=remotes},
+}
+
```

Comment:
Have you checked if the old Wickham `devtools` citation is still needed? Can we remove it?

Reply:
You are right. I removed it.